python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -1008,6 +1008,22 @@ def unhex(col: "ColumnOrName") -> Column:
 unhex.__doc__ = pysparkfuncs.unhex.__doc__
 
 
+def uniform(
+    min: Union[Column, int, float],
+    max: Union[Column, int, float],
+    seed: Optional[Union[Column, int]] = None,
+) -> Column:
+    if seed is None:
+        return _invoke_function_over_columns(
+            "uniform", min, max, lit(random.randint(0, sys.maxsize))
```

Comment:
`_invoke_function_over_columns` requires arguments be columns or column names

```suggestion
            "uniform", lit(min), lit(max), lit(random.randint(0, sys.maxsize))
```

Reply:
Thanks, this is done.